pom.xml

```diff
@@ -99,27 +99,57 @@
             </dependency>
 
             <dependency>
+                <groupId>com.google.inject</groupId>
+                <artifactId>guice</artifactId>
+                <version>4.0</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.google.inject.extensions</groupId>
+                <artifactId>guice-multibindings</artifactId>
+                <version>4.0</version>
+            </dependency>
+
+            <dependency>
```

Comment:
This is in the parent pom, and these are in the dependencyManagement section, which doesn't introduce deps directly, instead it just 1) specifies versions (and optionally scopes) for child poms that add a dependency on these libraries w/o a specific version, and 2) clearly resolves any conflicting dependency version requests.

In this case I'm using it for #2, since some code here wants 4.0, and some wants 4.0-beta5